From 05bea3d5ab1e6ff41173c1f5a03272d505374d62 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Tue, 28 Nov 2017 17:44:08 +0800
Subject: [PATCH 2903/5242] MLK-17014-2 ARM64: dts: imx8mq: move
 'display-subsystem' node to dtsi file

commit  c7905d6fdfcaa7668b0ac9586928b2976c1ed26d from
https://source.codeaurora.org/external/imx/linux-imx.git

It is better to put the 'display-subsystem' node definition
to 'fsl-imx8mq.dtsi' file.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 36dfb74..e8ae7c5 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -446,6 +446,11 @@
 		clock-names = "ipg", "per", "osc_per";
 	};
 
+	display-subsystem {
+		compatible = "fsl,imx-display-subsystem";
+		ports = <&dcss_disp0>;
+	};
+
 	irqsteer_dcss: irqsteer@32e2d000 {
 		compatible = "nxp,imx-irqsteer";
 		reg = <0x0 0x32e2d000 0x0 0x1000>;
-- 
1.7.9.5

